### DISCUSSION TITLE
winget and latest versions

### DISCUSSION THREAD

**Discussion by jonsagara** [JanDeDobbeleer/oh-my-posh#3460] [Troubleshoot] [answered]
Hi,

Do you publish every version to winget? I have seen in notification emails that oh-my-posh is up to v14.2.1, but winget still reports v14.0.2 as the latest version:

![image](https://user-images.githubusercontent.com/138811/217021937-ee17df09-5b50-4273-b004-759ae1b5faef.png)

Thank you,

Jon

**Comment by JanDeDobbeleer** [comment, marked as answer]
@jonsagara yes we do. Let me validate if they are stuck somewhere.

EDIT: all pr's are stuck on their end somehow. Will have a look.

**Comment by jonsagara** [comment]
I just checked, and the winget version is now up to `14.2.2`.

**Reply by JanDeDobbeleer** [reply]
@jonsagara yes, they fixed their CI issue

**Reply by jonsagara** [reply]
Thank you for looking into it!
